Ginger/Ginger/BusinessFlowPages_New/ActionsListViewPage.xaml.cs

```diff
@@ -183,6 +187,25 @@ private void listActions_ItemDropped(object sender, EventArgs e)
                     }
                     instance = null;
                 }
+                else if (droppedItem is ApplicationAPIModel || droppedItem is RepositoryFolder<ApplicationAPIModel>)
+                {
+                    ObservableList<ApplicationAPIModel> apiModelsList = new ObservableList<ApplicationAPIModel>();
+                    if (droppedItem is RepositoryFolder<ApplicationAPIModel>)
+                    {
+                        apiModelsList = (droppedItem as RepositoryFolder<ApplicationAPIModel>).GetFolderItems();
+                    }
+                    else
+                    {
+                        apiModelsList.Add(droppedItem as ApplicationAPIModel);
+                    }
+
+                    AddApiModelActionWizardPage APIModelWizPage = new AddApiModelActionWizardPage(mContext);
+                    APIModelWizPage.AAMList = apiModelsList;
+                    APIModelWizPage.Pages.RemoveAt(0);
+                    WizardWindow wizWindow = new WizardWindow(APIModelWizPage);
```

Comment:
what is the below for?
WizardWindow wizWindow = new WizardWindow(APIModelWizPage);
APIModelWizPage.GetCurrentPage().Page.WizardEvent(new WizardEventArgs(APIModelWizPage, EventType.Active));
wizWindow.Show();

only for showing the Wizard? if yes this is not the way, you should use:
WizardWindow.ShowWizard(new APIModelWizPage());

Reply:
Done.